2024-03-24 21:52:45.604[1711291965604] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Starting BotAplication using Java 17.0.10 with PID 2312 (/Users/dong1108/code/binance-futures-connector-java-main/target/classes started by dong1108 in /Users/dong1108/code/binance-futures-connector-java-main)
2024-03-24 21:52:45.606[1711291965606] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - No active profile set, falling back to 1 default profile: "default"
2024-03-24 21:52:45.638[1711291965638] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2024-03-24 21:52:45.638[1711291965638] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2024-03-24 21:52:46.093[1711291966093] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat initialized with port 8080 (http)
2024-03-24 21:52:46.099[1711291966099] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Initializing ProtocolHandler ["http-nio-8080"]
2024-03-24 21:52:46.100[1711291966100] | INFO  | restartedMain | o.a.catalina.core.StandardService    - Starting service [Tomcat]
2024-03-24 21:52:46.100[1711291966100] | INFO  | restartedMain | o.a.catalina.core.StandardEngine     - Starting Servlet engine: [Apache Tomcat/10.1.18]
2024-03-24 21:52:46.119[1711291966119] | INFO  | restartedMain | o.a.c.c.C.[Tomcat].[localhost].[/]   - Initializing Spring embedded WebApplicationContext
2024-03-24 21:52:46.120[1711291966120] | INFO  | restartedMain | o.s.b.w.s.c.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 480 ms
2024-03-24 21:52:46.303[1711291966303] | INFO  | restartedMain | o.s.b.d.a.OptionalLiveReloadServer   - LiveReload server is running on port 35729
2024-03-24 21:52:46.312[1711291966312] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Starting ProtocolHandler ["http-nio-8080"]
2024-03-24 21:52:46.321[1711291966321] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat started on port 8080 (http) with context path ''
2024-03-24 21:52:46.328[1711291966328] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Started BotAplication in 0.868 seconds (process running for 1.603)
2024-03-24 21:52:50.016[1711291970016] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:53:10.492[1711291990492] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711291990375&signature=7f890770603c9f07f175844b7a9b548b6798b6bf734dcecfe1cdfe107da9920c
2024-03-24 21:53:18.935[1711291998935] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711291998927&signature=e68c3cdc189707891d7e7c52159c32079348b75d21734d69ba61f543b7a0c0bb
2024-03-24 21:53:58.111[1711292038111] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:54:00.002[1711292040002] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:54:46.606[1711292086606] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Starting BotAplication using Java 17.0.10 with PID 2647 (/Users/dong1108/code/binance-futures-connector-java-main/target/classes started by dong1108 in /Users/dong1108/code/binance-futures-connector-java-main)
2024-03-24 21:54:46.607[1711292086607] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - No active profile set, falling back to 1 default profile: "default"
2024-03-24 21:54:46.631[1711292086631] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2024-03-24 21:54:46.632[1711292086632] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2024-03-24 21:54:47.066[1711292087066] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat initialized with port 8080 (http)
2024-03-24 21:54:47.071[1711292087071] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Initializing ProtocolHandler ["http-nio-8080"]
2024-03-24 21:54:47.073[1711292087073] | INFO  | restartedMain | o.a.catalina.core.StandardService    - Starting service [Tomcat]
2024-03-24 21:54:47.073[1711292087073] | INFO  | restartedMain | o.a.catalina.core.StandardEngine     - Starting Servlet engine: [Apache Tomcat/10.1.18]
2024-03-24 21:54:47.096[1711292087096] | INFO  | restartedMain | o.a.c.c.C.[Tomcat].[localhost].[/]   - Initializing Spring embedded WebApplicationContext
2024-03-24 21:54:47.096[1711292087096] | INFO  | restartedMain | o.s.b.w.s.c.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 464 ms
2024-03-24 21:54:47.264[1711292087264] | INFO  | restartedMain | o.s.b.d.a.OptionalLiveReloadServer   - LiveReload server is running on port 35729
2024-03-24 21:54:47.273[1711292087273] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Starting ProtocolHandler ["http-nio-8080"]
2024-03-24 21:54:47.280[1711292087280] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat started on port 8080 (http) with context path ''
2024-03-24 21:54:47.288[1711292087288] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Started BotAplication in 0.812 seconds (process running for 1.201)
2024-03-24 21:54:50.011[1711292090011] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:54:58.211[1711292098211] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292098109&signature=7eb2cb314fa1c6788754ee32870a4aebf0519af2ff81b873a91bf0a4832289a0
2024-03-24 21:55:03.980[1711292103980] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292103974&signature=23f1d1434f75d42189d52a2d90a6b4f87aa8ce107fcd1a0c7abfc13000977f28
2024-03-24 21:55:13.880[1711292113880] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:55:15.002[1711292115002] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:55:35.877[1711292135877] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292135874&signature=dceb750b86177b490edf38ac19daffe965533797d230630a97f5eb3efee83f99
2024-03-24 21:55:38.495[1711292138495] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292138493&signature=d77a7f031f497449da0d4c4bf926a513b7e802a19f5cd1c2276a74393b84ba85
2024-03-24 21:55:38.632[1711292138632] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:55:40.005[1711292140005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:55:50.141[1711292150141] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292150140&signature=0ef6d3296fac25d4b404c122fdbc5557b0d587d36e4a0d10836a32d8864dd07b
2024-03-24 21:55:50.267[1711292150267] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292150267&signature=fa0905683e450ceb73d905c3433f954bf26e5418e93f05717335c14f79c8b31a
2024-03-24 21:55:50.391[1711292150391] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:55:55.005[1711292155005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:55:55.006[1711292155006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292155006&signature=b7aa0183e5b026a2479042b352b22e5e329d2a60fc383fba9294a0a55aab3e18
2024-03-24 21:55:55.395[1711292155395] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292155395&signature=e22298806e302a3332f1ebb3d7eb397bab331ffe42c43d3f07c0b83c27fa4f05
2024-03-24 21:55:55.705[1711292155705] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:00.002[1711292160002] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:00.003[1711292160003] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292160002&signature=8cfdaad8bac9daa425f09c47add6c78dcec783b9bb602b632eb14cecccae977e
2024-03-24 21:56:00.128[1711292160128] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292160127&signature=9f3d5b3bd184e986cee87ae0b8c939b934ace32f06fb59681e5681baef911752
2024-03-24 21:56:00.239[1711292160239] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:05.005[1711292165005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:05.006[1711292165006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292165006&signature=046a75c1a72b14e6ffa948a2abc7e986c80a7297aaf15d9c60e24cf85a2e40e1
2024-03-24 21:56:05.431[1711292165431] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292165431&signature=23e3ad91ca057af67c8168038d62e4fa5f9b2331d135807b90cdbf2100bf6123
2024-03-24 21:56:05.540[1711292165540] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:10.005[1711292170005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:10.006[1711292170006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292170005&signature=d6516731bc90a8a365d7c0c4439725935915fba3d3d34b38c278609557725b52
2024-03-24 21:56:10.346[1711292170346] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292170345&signature=23ca842ba16845aedf2d1603c19546ee1d84ca8bbeb0f629db698e5cd31322d2
2024-03-24 21:56:10.655[1711292170655] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:14.948[1711292174948] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:14.949[1711292174949] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292174949&signature=0486833946b799eb54e11da6f87181e0a00cbff74520778a63aadd7627a1706c
2024-03-24 21:56:15.417[1711292175417] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292175416&signature=3b85fdbce6a244a1507f8f1785c63535a7a6c21801baf9d0c6a34852a5032ea1
2024-03-24 21:56:15.717[1711292175717] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:20.005[1711292180005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:20.005[1711292180005] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292180005&signature=93f67dad8d42ce9460999c9e80888559e4adcc76736f2debbadc40e26785c7e1
2024-03-24 21:56:20.323[1711292180323] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292180323&signature=af35738824b054e8ed5f3f42ba6dcf58221f6d37243e2b1e17942f19030b596b
2024-03-24 21:56:20.631[1711292180631] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:25.005[1711292185005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:25.006[1711292185006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292185005&signature=ebaf92d2bed6c0632cf515e63aff0fb9461646c30540fdb31316888b2df0a564
2024-03-24 21:56:25.342[1711292185342] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292185342&signature=a0e61ecd53dd24a32a5f4928320e5feda7577ff1fbcfc0002d0a3cb9f88fd8b7
2024-03-24 21:56:25.641[1711292185641] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:30.005[1711292190005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:30.006[1711292190006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292190005&signature=5d200362f0263a3a8cb9660eac1fa0a3000c70bcd466cf53c01b352bb48f7165
2024-03-24 21:56:30.127[1711292190127] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292190127&signature=579bdb427d5385b694ed8897a0d59ef1155a3910c88d8e56858acd30163bf88b
2024-03-24 21:56:30.464[1711292190464] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:35.002[1711292195002] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:35.003[1711292195003] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292195003&signature=5edc1ff89c89826531955cb95fa08397a97a5db03e91ee57b2997f10c33f0c4b
2024-03-24 21:56:35.379[1711292195379] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292195379&signature=8c38e7d358f8d3ac59ba9a72342a4f8c564e2dc1cc2324f3ed39f15f9e00a4e3
2024-03-24 21:56:35.683[1711292195683] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:40.005[1711292200005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:40.005[1711292200005] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292200005&signature=f433ecc34f9e7981a9b6e34371c826ae0dfa2c705fee6630d473dd1e7cb329db
2024-03-24 21:56:40.395[1711292200395] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292200395&signature=b23aa4c5c070b75f2937704e15045b9c292b6d9af6fc279b681b34fef6086a00
2024-03-24 21:56:40.505[1711292200505] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:45.005[1711292205005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:45.006[1711292205006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292205005&signature=90bb9c057c8fa2bee244b89c5f4eedbce39d52cfe30897c8015d850788c845b9
2024-03-24 21:56:45.310[1711292205310] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292205309&signature=53e5bf997505a6f7b6218613e2350787ce2f9002a13a06276220c695c9d26c13
2024-03-24 21:56:45.619[1711292205619] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:50.002[1711292210002] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:50.003[1711292210003] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292210003&signature=28fa669dadb3589e0ddd06c0395b71901e941e68f1ded7cf544260394fb28197
2024-03-24 21:56:50.123[1711292210123] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292210123&signature=e2f69f6375a3a76d964c9a9d3a942c312893c2fa2a3c352d1031c858cbbd44a7
2024-03-24 21:56:50.532[1711292210532] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:55.005[1711292215005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:56:55.006[1711292215006] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292215006&signature=de7c727f56cab7076d816c7e54c994d9f08990b84026749013ece7228acfbfee
2024-03-24 21:56:55.131[1711292215131] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292215131&signature=4db3715f1513d5f158bab20b90d61af8d4dd10bfe7c9a396e240dd83851d0d94
2024-03-24 21:56:55.449[1711292215449] | ERROR | scheduling-1 | o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task
org.json.JSONException: JSONArray[1] not found.
	at org.json.JSONArray.get(JSONArray.java:244)
	at org.json.JSONArray.getJSONObject(JSONArray.java:457)
	at com.binance.connector.futures.sheduled.SheduledTask.checkConditionCreateNewOrders(SheduledTask.java:68)
	at com.binance.connector.futures.sheduled.SheduledTask.callOrder(SheduledTask.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.runInternal(ScheduledMethodRunnable.java:130)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.lambda$run$2(ScheduledMethodRunnable.java:124)
	at io.micrometer.observation.Observation.observe(Observation.java:499)
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:124)
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54)
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:96)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:842)
2024-03-24 21:56:58.093[1711292218093] | INFO  | File Watcher | o.s.b.d.a.LocalDevToolsAutoConfiguration$RestartingClassPathChangeChangedEventListener - Restarting due to 1 class path change (0 additions, 0 deletions, 1 modification)
2024-03-24 21:56:58.100[1711292218100] | INFO  | Thread-7   | o.a.coyote.http11.Http11NioProtocol  - Stopping ProtocolHandler ["http-nio-8080"]
2024-03-24 21:56:58.164[1711292218164] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Starting BotAplication using Java 17.0.10 with PID 2647 (/Users/dong1108/code/binance-futures-connector-java-main/target/classes started by dong1108 in /Users/dong1108/code/binance-futures-connector-java-main)
2024-03-24 21:56:58.164[1711292218164] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - No active profile set, falling back to 1 default profile: "default"
2024-03-24 21:56:58.260[1711292218260] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat initialized with port 8080 (http)
2024-03-24 21:56:58.261[1711292218261] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Initializing ProtocolHandler ["http-nio-8080"]
2024-03-24 21:56:58.261[1711292218261] | INFO  | restartedMain | o.a.catalina.core.StandardService    - Starting service [Tomcat]
2024-03-24 21:56:58.261[1711292218261] | INFO  | restartedMain | o.a.catalina.core.StandardEngine     - Starting Servlet engine: [Apache Tomcat/10.1.18]
2024-03-24 21:56:58.266[1711292218266] | INFO  | restartedMain | o.a.c.c.C.[Tomcat].[localhost].[/]   - Initializing Spring embedded WebApplicationContext
2024-03-24 21:56:58.267[1711292218267] | INFO  | restartedMain | o.s.b.w.s.c.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 101 ms
2024-03-24 21:56:58.309[1711292218309] | INFO  | restartedMain | o.s.b.d.a.OptionalLiveReloadServer   - LiveReload server is running on port 35729
2024-03-24 21:56:58.312[1711292218312] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Starting ProtocolHandler ["http-nio-8080"]
2024-03-24 21:56:58.314[1711292218314] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat started on port 8080 (http) with context path ''
2024-03-24 21:56:58.317[1711292218317] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Started BotAplication in 0.172 seconds (process running for 132.291)
2024-03-24 21:56:58.317[1711292218317] | INFO  | restartedMain | o.s.b.d.a.ConditionEvaluationDeltaLoggingListener - Condition evaluation unchanged
2024-03-24 21:57:00.005[1711292220005] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:57:00.013[1711292220013] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292220011&signature=9377bebaad3f3597ba15b2adba61be4a60363ad9bcc9f9e0aceab85e1a36ed50
2024-03-24 21:57:00.568[1711292220568] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292220568&signature=b44ebe168bca4fc85b95703779f9e04f514a48686b63ae37b1831a82b34f07d6
2024-03-24 21:57:00.876[1711292220876] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - POST https://testnet.binancefuture.com/fapi/v1/order?symbol=BTCUSDT&side=SELL&type=LIMIT&timeInForce=GTC&quantity=0.01&price=67311.8&timestamp=1711292220875&signature=20c0d36be793dbb9352f67bfaf2d2601a0d53b0c6a4a2ff5b666cb1d4cf43108
2024-03-24 21:57:01.284[1711292221284] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  - {"orderId":3739106194,"symbol":"BTCUSDT","status":"NEW","clientOrderId":"5KjZOZcyGSlS9HC33L6tf9","price":"67311.80","avgPrice":"0.00","origQty":"0.010","executedQty":"0.000","cumQty":"0.000","cumQuote":"0.00000","timeInForce":"GTC","type":"LIMIT","reduceOnly":false,"closePosition":false,"side":"SELL","positionSide":"BOTH","stopPrice":"0.00","workingType":"CONTRACT_PRICE","priceProtect":false,"origType":"LIMIT","priceMatch":"NONE","selfTradePreventionMode":"NONE","goodTillDate":0,"updateTime":1711292221232}
2024-03-24 21:57:08.448[1711292228448] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Starting BotAplication using Java 17.0.10 with PID 3174 (/Users/dong1108/code/binance-futures-connector-java-main/target/classes started by dong1108 in /Users/dong1108/code/binance-futures-connector-java-main)
2024-03-24 21:57:08.450[1711292228450] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - No active profile set, falling back to 1 default profile: "default"
2024-03-24 21:57:08.474[1711292228474] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2024-03-24 21:57:08.474[1711292228474] | INFO  | restartedMain | o.s.b.d.e.DevToolsPropertyDefaultsPostProcessor - For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2024-03-24 21:57:08.926[1711292228926] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat initialized with port 8080 (http)
2024-03-24 21:57:08.932[1711292228932] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Initializing ProtocolHandler ["http-nio-8080"]
2024-03-24 21:57:08.933[1711292228933] | INFO  | restartedMain | o.a.catalina.core.StandardService    - Starting service [Tomcat]
2024-03-24 21:57:08.933[1711292228933] | INFO  | restartedMain | o.a.catalina.core.StandardEngine     - Starting Servlet engine: [Apache Tomcat/10.1.18]
2024-03-24 21:57:08.953[1711292228953] | INFO  | restartedMain | o.a.c.c.C.[Tomcat].[localhost].[/]   - Initializing Spring embedded WebApplicationContext
2024-03-24 21:57:08.954[1711292228954] | INFO  | restartedMain | o.s.b.w.s.c.ServletWebServerApplicationContext - Root WebApplicationContext: initialization completed in 479 ms
2024-03-24 21:57:09.125[1711292229125] | INFO  | restartedMain | o.s.b.d.a.OptionalLiveReloadServer   - LiveReload server is running on port 35729
2024-03-24 21:57:09.133[1711292229133] | INFO  | restartedMain | o.a.coyote.http11.Http11NioProtocol  - Starting ProtocolHandler ["http-nio-8080"]
2024-03-24 21:57:09.141[1711292229141] | INFO  | restartedMain | o.s.b.w.e.tomcat.TomcatWebServer     - Tomcat started on port 8080 (http) with context path ''
2024-03-24 21:57:09.149[1711292229149] | INFO  | restartedMain | c.b.connector.futures.BotAplication  - Started BotAplication in 0.835 seconds (process running for 1.196)
2024-03-24 21:57:10.008[1711292230008] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:57:10.024[1711292230024] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292230011&signature=02bcb540c2af1687c7a3b31dd2a32a44b81ba8eea2064aa20e902f54ea04a60f
2024-03-24 21:57:10.611[1711292230611] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292230611&signature=ca740bcfc14986a7fd3c676e7405e919b7bd97f79329ef4e172a5f4621de8892
2024-03-24 21:58:00.003[1711292280003] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:58:02.643[1711292282643] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/userTrades?symbol=BTCUSDT&limit=1&timestamp=1711292282639&signature=e6a2d9750bfd6947c66bd7177cf129d2ee2c089d178eaa5332b9f89aae84abcb
2024-03-24 21:58:04.341[1711292284341] | INFO  | scheduling-1 | c.b.c.f.client.utils.RequestHandler  - GET https://testnet.binancefuture.com/fapi/v1/openOrders?symbol=BTCUSDT&limit=1&timestamp=1711292284336&signature=ea46b04488c40ee5a27951b35e1c3f83e9ce089abda029dacac660d86f3ac9b2
2024-03-24 21:58:30.000[1711292310000] | INFO  | scheduling-1 | c.b.c.futures.sheduled.SheduledTask  -  ****** Begin job ****** 
2024-03-24 21:58:41.590[1711292321590] | INFO  | File Watcher | o.s.b.d.a.LocalDevToolsAutoConfiguration$RestartingClassPathChangeChangedEventListener - Restarting due to 1 class path change (0 additions, 0 deletions, 1 modification)
